_profiles/Protein/0.10-DRAFT-2020_04_02.html

```diff
@@ -0,0 +1,418 @@
+---
+name: Protein
+
+previous_version: 0.8-DRAFT-2019_05_08
+previous_release: 
+
+
+status: revision # Change to revision if you are working on an update
+spec_type: Profile
+group: proteins
+use_cases_url: /useCases/Protein/
+cross_walk_url: https://docs.google.com/spreadsheets/d/1UbTy5OACa08ZTX9yk35AaVRUS6YBm1jBqwgG-Y41KGc
+gh_tasks: https://github.com/BioSchemas/bioschemas/labels/type%3A%20Protein
+live_deploy: /liveDeploys/
+
+parent_type: Protein
+hierarchy:
+- Thing
+- BioChemEntity
+- Protein
+
+# spec_info content generated using GOWeb
+# DO NOT MANUALLY EDIT THE CONTENT
+spec_info:
+  title: Protein
+  subtitle: Bioschemas profile describing a Protein in Life Sciences.
+  description: 'This Protein profile specification presents the most common properties used for a 
```

Comment:
I've added formatting into the spreadsheet. Please regenerate using GoWEB

Reply:
Done